userspace/libsinsp/examples/CMakeLists.txt

```diff
@@ -9,3 +9,8 @@ add_executable(sinsp-example
 target_link_libraries(sinsp-example
 	sinsp
 )
+
+if (APPLE AND NOT MINIMAL_BUILD)
+	# Needed when linking libcurl
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -framework Foundation -framework SystemConfiguration")
```

Comment:
We need this when linking libcurl on macos (ie: when we are not in MINIMAL_BUILD)

Reply:
Note however that only MINIMAL_BUILD will actually runtime work for macos; ie: you can build the full build, but an exception will be thrown at runtime.
We only support MINIMAL_BUILD on macos indeed :)

Reply:
Btw i will leave this here to allow a full build, even if full build is not fully supported atm.